testing/util.py

```diff
@@ -86,7 +87,7 @@ def cmd_output_mocked_pre_commit_home(*args, **kwargs):
 
 
 def platform_supports_pcre():
-    output = cmd_output('grep', '-P', "name='pre", 'setup.py', retcode=None)
+    output = cmd_output(GREP, '-P', "name='pre", 'setup.py', retcode=None)
```

Comment:
This is not related to this PR, but the tests were failing on my mac and I just quickly changed it. Let me know if you want me to take it out.